app/renderer/components/Session/CapabilityControl.js

```diff
@@ -7,24 +7,24 @@ import { INPUT } from '../AntdTypes';
 
 const {dialog} = remote;
 
-
 export default class NewSessionForm extends Component {
 
-  getLocalFilePath (success) {
-    dialog.showOpenDialog((filepath) => {
-      if (filepath) {
-        success(filepath);
+  async getLocalFilePath (success) {
+    await dialog.showOpenDialog({
+      properties: ['openFile'],
+    }).then((files) => {
+      if (files.filePaths) {
+        success(files.filePaths);
       }
-    });
-    this.handleSetType = this.handleSetType.bind(this);
```

Comment:
don't we need this handler anymore?

Reply:
Seems no, we probably do not need it. `this.handleSetType` was `nil` here so `.bind` raised an error